packages/app/src/components/button/outline/index.tsx

```diff
@@ -109,13 +109,20 @@ const Renewal = function <T = null>({
     onClick(data as T);
   }, [data, onClick]);
 
+  const normalStyle = `bg-thm-${cs} hover:bg-thm-on-${cs}-slight text-thm-on-${cs} border-thm-on-${cs}-low`;
+  const olStyle = `bg-thm-background hover:bg-thm-on-background-slight text-thm-${cs} border-thm-${cs}`;
+
+  let buttonStyle = normalStyle;
+  if (cs === 'primary') {
+    buttonStyle = olStyle;
+  }
```

Comment:
If you need this change, there is most likely a design problem.
What is the design when cs is set as primary in normalStyle?